setup/borg/cli.md

```diff
@@ -121,87 +140,102 @@ After adding the repo, you can copy the repository location using the icon in th
 
 To initialize the remote repo, use the commands below. Replace the `ssh://` URL with your actual full repository address given in the control panel. The repository URL will be kept as `BORG_REPO` variable to avoid typing it each time.
 
 {% tabs init %}
 
 {% tab init Borg 1.x %}
+
 ```shell
 $ export BORG_REPO=ssh://...
 $ borg init -e repokey-blake2
 ```
+
 {% endtab %}
 
 {% tab init Borg 2.x %}
+
 ```shell
 $ export BORG_REPO=ssh://...
 $ borg benchmark cpu  # optional: find fastest encryption algorithm
 $ borg rcreate --encryption=repokey-blake2-aes-ocb
 ```
+
 **Note**: Encryption speed can vary depending on CPU and if a virtual machine is used. So it's best to benchmark it and choose the fastest combination. More [here](https://borgbackup.readthedocs.io/en/2.0.0b4/usage/rcreate.html#choosing-an-encryption-mode).
 {% endtab %}
 
 {% endtabs %}
 
+All those commands will ask you for a passphrase to protect the repository key. Make it nice and long and keep it somewhere safe.
 
-All those commands will ask you for a passphrase to protect the repository key. Make it nice and long and keep it somewhere safe. 
-
-After the command is done, you have your remote backup repository set up and ready to accept files. You can add some test files from your *Documents* folder with this commands:
+After the command is done, you have your remote backup repository set up and ready to accept files. You can add some test files from your _Documents_ folder with this commands:
 
 {% tabs create %}
 
 {% tab create Borg 1.x %}
+
 ```shell
 $ borg create ::my-archive-1 ~/Documents
 ```
+
 {% endtab %}
 
 {% tab create Borg 2.x %}
+
 ```shell
 $ borg create my-archive-1 ~/Documents
 ```
+
 {% endtab %}
 
 {% endtabs %}
 
 `my-archive-1` is the name of the snapshot. It can be anything, but mostly people use the hostname and a timestamp. E.g. `tim-macbook-2018-10-10`. Archives allow you to access different versions of your files. This can be useful if a file was deleted by accident or encrypted by a cryptolocker.
 
 ## Step 6 – Set up Borgmatic for Regular Backups
+
 It would be quite cumbersome to manually run Borg every few hours and keep coming up with new Snapshot names. That's why we will use Borgmatic to do this job for us.
 
 Let's start by creating a default Borgmatic folder:
+
 ```
 $ mkdir -p ~/.config/borgmatic
 ```
 
 Then open the main config file with your favorite text editor. I'll use `nano`, but `vi`, Sublime or Atom will work too.
+
 ```
 $ nano ~/.config/borgmatic/config.yaml
 ```
 
 If you keep your repos with BorgBase you can copy a pre-made Borgmatic confirguration from the **Setup** page. Or start with this version and fill in the blanks:
 
-```
+{% tabs borgmatic_config %}
+
+{% tab borgmatic_config Borgmatic >= 1.8 %}
```

Comment:
Please add a note and link (to the changelog) regarding Borgmatic config format. So it's clear why we have 2 versions.